ts/annotations/annotations.src.ts

```diff
@@ -1640,7 +1670,14 @@ extend(chartProto, /** @lends Highcharts.Chart# */ {
         this.plotBoxClip.attr(this.plotBox);
 
         this.annotations.forEach(function (annotation): void {
+            const deferDiff = annotation.deferTime - annotation.durationTime;
             annotation.redraw();
+            annotation.graphic.animate({
+                opacity: 1
+            }, {
+                defer: deferDiff === 0 ? annotation.deferTime : deferDiff,
```

Comment:
Are you sure this ternary op is needed? It's quite the difference between setting `defer=200` (and below 200) and `defer=201` (and above):
- defer = 200 -> after 200ms run 200ms animation
- defer = 201 -> after 1ms run 200ms animation

Reply:
After internal discussion - we back to the previous implementation, so:

```js
    durationTime = Math.min(deferTime, 200);
    stackTotalGroup.animate({
        opacity: 1
    }, {
        duration: durationTime,
        defer: deferTime - durationTime
    });
```

which means that: 

```js
//duration is set to 200
if (defer < duration) {
defer = 0
duration = defer
}
```